bug: passing default path in router causes error

As per https://github.com/yoshuawuyts/sheet-router/issues/21 when passing a default path to the router, an error is thrown:
```txt
assert.js:59Uncaught AssertionError: createTree must be a function
```
This only happens in `choo`. Should probably be an argument order error

--- a/index.js
+++ b/index.js
@@ -179,9 +179,9 @@ function choo () {
   }
 
   // register all routes on the router
-  // [obj|fn] -> null
-  function router (cb) {
-    _router = sheetRouter(cb)
+  // (str?, [fn|[fn]]) -> obj
+  function router (defaultRoute, cb) {
+    _router = sheetRouter(defaultRoute, cb)
     return _router
   }
 

